src/Components/HomePage/Homepage.js

```diff
@@ -2,58 +2,118 @@ import React from 'react'
 import './Homepage.scss'
 import { ActivityCards }  from '../ActivityCards/ActivityCards'
 import { CgUserlane } from "react-icons/cg";
-import { FaServer, FaGithub} from "react-icons/fa";
 import { Link } from 'react-router-dom';
 
-export function Homepage() {
+
+
+export const Homepage = (props) => {
+
     return (
         <div className='homepage-container'>
 
-          <nav className="nav-bar">
+          <div className="homepage-inner-container">
+
             <div className="app-logo">
               <CgUserlane className="logo"/>
             </div>
 
-            <ul className="nav-links">
-              <li className="link">About</li>
-              <li className="link">Developers</li>
-              <li className="link"></li>
-            </ul>
-          </nav>
+            <nav className="nav-bar">
+              <ul className="nav-links">
+              <Link
+                  id="About"
+                  className="link"
+                  to="/about">
+                <li 
+                  id="About"
+                  name="About"
+                  >About</li>
```

Comment:
I wonder if there is a way we can iterate over these since the data seems pretty consistent for all of them. Granted, I have never mapped router components but its worth a shot!

Reply:
I bet there is a way using the event !